JASP-Engine/JASP/R/ttestpairedsamples.R

```diff
@@ -459,4 +456,21 @@ TTestPairedSamples <- function(jaspResults, dataset = NULL, options, ...) {
   return(p)
 }
 
+.ttestPairedGetHypothesisFootnote <- function(hypothesis, pairs) {
+  directionNote <- if (hypothesis == "groupTwoGreater") gettext("less") else gettext("greater")
```

Comment:
It's better not to translate `"less"` and `" greater"`  separately; it may well be possible that these words are translated differently as standalone words than when they are translated within the context of this sentence. This is not something we've done from the start, but it's something we should start doing.

So basically complete sentences > sentence with word substitutions, see also https://www.gnu.org/software/gettext/manual/gettext.html#Entire-sentences

(@FransMeerhoff you probably will want to update your translation guide and maybe make a post about the nuisances related to translations)